react/features/recording/components/LiveStream/BroadcastList.web.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { translate } from '../../../base/i18n';
+
+import BroadcastListItem from './BroadcastListItem';
+
+/**
+ * A React Component for display information for a live stream.
+ *
+ * @extends Component
+ */
+class BroadcastList extends Component {
+    /**
+     * {@code BroadcastList} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        broadcasts: PropTypes.array,
+        onClick: PropTypes.func,
+        t: PropTypes.func
```

Comment:
is `t` a special thing or a convention or something?  why not a more descriptive name?

Reply:
i see `title` in `BroadcastListItem` below, is that what this is? can we rename it to `title`?

Reply:
t is what's passed in by i18next translate. I could alias it but I'd rather not. https://react.i18next.com/components/i18n-render-prop.html